packages/date-time-picker/src/vaadin-date-time-picker.d.ts

```diff
@@ -93,7 +93,24 @@ export interface DateTimePickerEventMap extends DateTimePickerCustomEventMap, HT
  *
  * See [Styling Components](https://vaadin.com/docs/latest/styling/styling-components) documentation.
  *
+ * ### Change events
+ *
+ * Depending on the nature of the value change that the user attempts to commit e.g. by pressing Enter,
+ * the component can fire either a `change` event or an `unparsable-change` event:
+ *
+ * Value change             | Event
+ * :------------------------|:------------------
+ * empty => parsable        | change
+ * empty => unparsable      | unparsable-change
+ * parsable => empty        | change
+ * parsable => parsable     | change
+ * parsable => unparsable   | change
+ * unparsable => empty      | unparsable-change
+ * unparsable => parsable   | change
+ * unparsable => unparsable | unparsable-change
```

Comment:
I'd suggest including the `incomplete` scenario in this list separately.

Reply:
Done.